merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/types/Duration.java

```diff
@@ -148,6 +149,7 @@ private Duration(final long durationInMicroseconds) {
   public static final Duration MINUTE = SECOND.times(60);
   /** One hour (h), equal to 60m. */
   public static final Duration HOUR = MINUTE.times(60);
+  public static final Duration DAY = HOUR.times(24);
```

Comment:
I agree that there are multiple definitions of a day that could be useful, but the usual definition of day is so ubiquitous that omitting it has always confused me. I think if you ask anyone, even a rover planner, how long a day is, they will say "24 hours", not "are we on Mars?". Even when martian sols or sidereal days are relevant, I don't think anyone will blindly assume that `Duration.DAY` refers to anything other than a synodic day, and if we don't define it, all of our users will. The same applies to a week but I forgot to add it.

I agree that we shouldn't provide months or years, but not for the stated reasons. Days have multiple definitions, but months arguably have no definition as a pure duration. It would be problematic to define a month as 30 days, not because there are alternate definitions, but because it is wrong.

Reply:
Given that no duration longer than a second has a reliable definition (leap seconds result in minutes with 59, 60, or 61 seconds, which propagate up to affect the containing hour and day. Days may have 23, 24, or 25 hours around daylight savings boundaries), let's not let the edge cases get in the way of providing useful default durations. 

I think having DAY as shorthand for 24 hours follows the principle of least surprise. If someone needs a different definition, they can define one. 

Longer term, I'd like to look into moving some of the convenience functionality out of the protocol, whose primary (and, ideally, sole) responsibility is to define the methods and data structures that the merlin driver uses to interrogate a mission model